configs/Nanostack_ETH_config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Instead of having this new file, could we modify the eth_ipv6.json and use that for this purpose? One config less.